packages/rest/src/rest.server.ts

```diff
@@ -281,7 +282,14 @@ export class RestServer extends Context implements Server, HttpServerLike {
     // See https://github.com/strongloop/loopback-next/issues/433
     if (this._httpHandler) return;
 
-    this._httpHandler = new HttpHandler(this);
+    const router = this.getSync(RestBindings.ROUTER, {optional: true});
+    let routingTable: RoutingTable;
+    if (router) {
```

Comment:
Can we move this `if` to `RoutingTable` constructor so that RestServer can be simplified as follows:

```ts
const routingTable = new RoutingTable(router);
```